cypress/integration/site-search/index.spec.ts

```diff
@@ -21,7 +21,5 @@ describe("Docs website search", () => {
       .type("{enter}");
     cy.wait("@searchRequest");
     cy.get('[data-cy="paste-docsearch-hits"] h2').should("have.length.above", 0);
-    cy.get('[data-cy="paste-docsearch-hits"] ul').should("have.length.above", 0);
-    cy.get('[data-cy="paste-docsearch-hits"] li').should("have.length.above", 0);
```

Comment:
This would only be successful if the search result returned a page result. If the search query returns only github discussions it would fail, as discussion don't have section. 

The page sections are listed as lists under the page title.

<img width="610" alt="image" src="https://github.com/twilio-labs/paste/assets/368249/1f76d4c6-dd42-4ea3-af52-686733b8db33">